R/staging.txt

```diff
@@ -0,0 +1 @@
+http://biovirologia/

```

Comment:
Seems like the URL should be https://aimergdiaz.github.io/Bio-Virologia/, right ?